x-pack/plugin/sql/sql-cli/build.gradle

```diff
@@ -51,6 +51,7 @@ dependencyLicenses {
  * can be easily shipped around and used.
  */
 jar {
+    dependsOn configurations.compile, configurations.runtime
```

Comment:
There's no need to list both `compile` and `runtime` here given that `runtime` _includes_ compile dependencies. We should use `runtime` only and do the same in the `from()` below. While we're at it, we should use `runtimeClasspath` instead given that `runtime` is deprecated.